docker.d/init_worker.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+# == START: this is what we need to configure ==
+test AUTH0_CLIENT_ID
+test AUTH0_CLIENT_SECRET
+# == END:   this is what we need to configure ==
+
+case $ENV in
+  dev|stage)
+    export API_ROOT="https://admin-stage.balrog.nonprod.cloudops.mozgcp.net/api"
+    ;;
+  prod)
+    export API_ROOT="https://aus4-admin.mozilla.org/api"
```

Comment:
Hm. It looks like `init_worker.sh` and `worker.yml` differ between balrogscript and treescript, but the others are the same. When we go to the monorepo, we likely need one of the following solutions:

- a script-specific `init_worker.sh`,
- a shared `init_worker.sh` with a bunch of `if` blocks
- a shared simple `init_worker.sh`. we could move config items to another file, e.g. the `worker.yml` json-e or a `balrogscript_prod_env.sh` file that we can source.

Reply:
i found worker related configs differ too much to be able to abstract anything. every configuration of worker is 99% different. the only thing that i could keep the same is `docker.d/init.sh` and `docker.d/scriptworker.yml`.

Once I'm done with the configuration you can look at different `docker.d/worker.yml` and `docker.d/init_worker.sh` and you will see that not much is similar.